src/dvc_task/app/filesystem.py

```diff
@@ -5,13 +5,16 @@
 
 from celery import Celery
 from kombu.message import Message
+from kombu.transport.filesystem import Channel
```

Comment:
```suggestion
from kombu.transport.filesystem import Channel  # type: ignore
```

The 3rd party celery/kombu type stubs don't include this import, need to ignore it to fix linter errors